.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -73,10 +106,17 @@ steps:
       -e ARCHITECTURE=${ARCHITECTURE} \
       ${CONTAINER_IMAGE} make -f packer.mk test-building-vhd
     displayName: Run VHD cache test
+
+# Write release notes only if the mode is Default Gen 1 or it's sigMode Gen 2 (built sig is for production)
   - task: PublishPipelineArtifact@0
     inputs:
         artifactName: 'vhd-release-notes-${{ parameters.artifactName }}'
         targetPath: 'release-notes.txt'
+    condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), 
```

Comment:
What is the purpose to add the condition for publishing release notes? We can generate release notes for check-in pr and sig mode to validate whether it is expected.

Reply:
Removed.